### ISSUE TITLE
Quotes in NSIS arguments cause the build process to error out

### ISSUE THREAD

**Issue by rGunti** [electron-userland/electron-builder#5105]
* **Version**: 22.6.0

* **Electron Version**: 8.2.5
* **Electron Type (current, beta, nightly)**: current

* **Target**: Windows, portable + NSIS

I am currently building an electron app and wanted to build and publish it as a portable Windows EXE. When building it, I was always greeted with this error when building for anything involving NSIS:

```
VIAddVersionKey expects 2-3 parameters, got 5.
Usage: VIAddVersionKey [/LANG=lang_id] keyname value
```

I couldn't find anything when using all my search machines of choice and when after I discovered the reference for "VIAddVersionKey" (https://nsis.sourceforge.io/Reference/VIAddVersionKey) it his me: I have defined my alias in the autor section of my package.json in quotes like this:

```
  "author": {
    "name": "Raphael \"rGunti\" Guntersweiler",
    // ...
  },
```

So I replaced them with `'` instead and it worked! So my conclusion from that is: Somewhere in the build process, arguments for NSIS don't get properly escaped before being sent through. I could replicate this on Windows and on macOS when building for Windows portable.

**tl;dr**: Using quotes in application metadata confuses NSIS because they get passed without escaping.

**Comment by rGunti** [comment]
P.S.: Full Output from Build:

```
  • electron-builder  version=22.6.0 os=10.0.19041
  • loaded configuration  file=D:\Projects\twitch-clip-grabber\electron-builder.json
  • writing effective config  file=release\builder-effective-config.yaml
  • packaging       platform=win32 arch=x64 electron=8.2.5 appOutDir=release\win-unpacked
  • building        target=portable file=release\Sir Grabbington 0.1.0.exe archs=x64
  ⨯ C:\Users\rapha\AppData\Local\electron-builder\Cache\nsis\nsis-3.0.4.1\Bin\makensis.exe exited with code ERR_ELECTRON_BUILDER_CANNOT_EXECUTE
Output:

Output:
Command line defined: "APP_ID=com.electron.sir-grabbington"
Command line defined: "APP_GUID=..."
Command line defined: "UNINSTALL_APP_KEY=..."
Command line defined: "PRODUCT_NAME=My App Name"
Command line defined: "PRODUCT_FILENAME=My App Name"
Command line defined: "APP_FILENAME=my-app-name"
Command line defined: "APP_DESCRIPTION=My Description"
Command line defined: "VERSION=0.1.0"
Command line defined: "PROJECT_DIR=D:\Projects\my-app-name"
Command line defined: "BUILD_RESOURCES_DIR=D:\Projects\my-app-name\build"
Command line defined: "APP_PACKAGE_NAME=my-app-name"
Command line defined: "APP_64=D:\Projects\my-app-name\release\my-app-name-0.1.0-x64.nsis.7z"
Command line defined: "APP_64_NAME=my-app-name-0.1.0-x64.nsis.7z"
Command line defined: "APP_64_HASH=..."
Command line defined: "COMPANY_NAME=Raphael "rGunti" Guntersweiler"
Command line defined: "APP_PRODUCT_FILENAME=MyAppName"
Command line defined: "APP_INSTALLER_STORE_FILE=MyAppName-updater\installer.exe"
Command line defined: "COMPRESSION_METHOD=7z"
Command line defined: "REQUEST_EXECUTION_LEVEL=user"
Command line defined: "UNPACK_DIR_NAME=..."
Command line defined: "ESTIMATED_SIZE=642391"
Command line defined: "COMPRESS=auto"
Command line defined: "UNINSTALLER_OUT_FILE"

Error output:

Error output:
VIAddVersionKey expects 2-3 parameters, got 5.
Usage: VIAddVersionKey [/LANG=lang_id] keyname value
  stackTrace=
```
